lib/active_merchant/billing/gateways/micropayment.rb

```diff
@@ -81,11 +84,12 @@ def add_invoice(post, money, options)
         post[:project] = options[:project] || "sprdly"
       end
 
-      def add_payment_method(post, payment_method)
+      def add_payment_method(post, payment_method, options={})
         post[:firstname] = payment_method.first_name
         post[:surname] = payment_method.last_name
         post[:number] = payment_method.number
-        post[:cvc2] = payment_method.verification_value
+        post[:recurring] = options[:recurring] == true ? 1 : 0
+        post[:cvc2] = payment_method.verification_value unless post[:recurring] == 1
```

Comment:
Would things work with the `unless` clause?  So we always pass the `verification_value`?